test/explorer-client.js

```diff
@@ -0,0 +1,39 @@
+import { assert } from 'chai';
+const url = require('url');
+const fetch = require('node-fetch');
+
+import {
+  BlockExplorerClient,
+  BLOCK_EXPLORERS,
+  EXPLORER_SUFFIXES,
```

Comment:
Only `BlockExplorerClient` is used in this file. Can get rid of the other two imports

Reply:
👍